internal/syncer/connector/github.go

```
@@ -75,7 +75,7 @@ func (c *GitHubConnector) Validate(ctx context.Context) error {
 		return fmt.Errorf("github connector is nil")
 	}
 	if c.owner == "" {
-		return fmt.Errorf("Invalid connector settings: 'repo_owner' must be provided")
+		return fmt.Errorf("Invalid connector settings: 'repository_owner' must be provided")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)